framework/LoginDialog/LoginDialogWindow.bbj

```diff
@@ -26,6 +25,51 @@ class public LoginDialogWindow
         #content!.addPanelStyle("loginWidgetContentPanelStyle")
     methodend 
 
+    method public Boolean canSubmit()
```

Comment:
This is the wrong way of doing it. We don't want our inputfield.bbj tied to the only login module, that's why fields like username, repeatpassword shouldn't be here. I think we need a call, so that I can explain it you.